pkg/crd/operator/v1alpha1/configuration.go

```diff
@@ -162,13 +162,10 @@ type KubermaticUserClusterConfiguration struct {
 	NodePortRange string `json:"nodePortRange,omitempty"`
 	// Monitoring can be used to fine-tune to in-cluster Prometheus.
 	Monitoring KubermaticUserClusterMonitoringConfiguration `json:"monitoring,omitempty"`
-	// DisableAPIServerEndpointReconciling can be used to toggle the `--endpoint-reconciler-type` flag for
-	// the Kubernetes API server.
-	DisableAPIServerEndpointReconciling bool `json:"disableApiserverEndpointReconciling,omitempty"`
 	// EtcdVolumeSize configures the volume size to use for each etcd pod inside user clusters.
 	EtcdVolumeSize string `json:"etcdVolumeSize,omitempty"`
-	// APIServerReplicas configures the replica count for the API-Server deployment inside user clusters.
-	APIServerReplicas *int32 `json:"apiserverReplicas,omitempty"`
+	// DefaultClusterOverridesConfigMap is the name of the configmap with default components override settings.
+	DefaultClusterOverridesConfigMap string `json:"defaultClusterOverridesConfigMap"`
```

Comment:
Same as per previous comments: we could avoid breaking backward compatibility and use file instead of ConfigMap name as a contract.